pymeasure/instruments/oxfordinstruments/itc503.py

```diff
@@ -177,10 +206,14 @@ def __init__(self, resourceName, clear_buffer=True, **kwargs):
         if clear_buffer:
             self.adapter.connection.clear()
 
+        self._T_RANGE[0] = min_temperature
+        self._T_RANGE[1] = max_temperature
+
     def wait_for_temperature(self, error=0.01, timeout=3600,
                              check_interval=0.5, stability_interval=10,
                              thermalize_interval=300,
-                             should_stop=lambda: False):
+                             should_stop=lambda: False,
+                             max_comm_errors=0):
```

Comment:
I find a value of "0" meaning "infinity" confusing. Maybe use `None` instead -- see comment below.

Reply:
I agree; fixed this issue.